src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import FONTFAMILY from "../../../variables/font_family";
+
+const Input = ({ onEditComplete, defaultValue }) => {
+  const ref = useRef(null);
+
+  useEffect(() => {
+    let value = null;
+    if (defaultValue == null) {
+      value = "";
+    } else {
+      value = defaultValue;
+    }
+
+    ref.current.focus();
+  }, []);
+
+  const handleEnter = (e) => {
+    if (e.key == "Enter") {
+      e.preventDefault();
+      onEditComplete();
+    }
+  };
+
+  return (
+    <StyleldInput
+      ref={ref}
+      onBlur={onEditComplete}
+      onKeyPress={(e) => handleEnter(e)}
```

Comment:
```suggestion
      onBlur={onEditComplete}
      onKeyPress={handleEnter}
```
一応これでもいけると思います（アロー関数で書かずとも第一引数にイベントオブジェクトが渡されるので）

Reply:
ちなみに、useEffect内でrefに対してaddEventListnerを設定するというやり方もあるので、そちらも確認しておいていただけると助かります！